app/services/monitoring/DeclarationBlockBuilder.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.monitoring
+
+import models.api.{Address, FormerName, Name, VatScheme}
+import play.api.libs.json.{JsObject, Json}
+import uk.gov.hmrc.http.InternalServerException
+import utils.JsonUtils.{jsonObject, _}
+
+import javax.inject.Singleton
+
+@Singleton
+class DeclarationBlockBuilder {
+
+  def buildDeclarationBlock(vatScheme: VatScheme): JsObject = {
+    (vatScheme.applicantDetails, vatScheme.confirmInformationDeclaration) match {
+      case (Some(applicantDetails), Some(declaration)) =>
+        jsonObject(
+          "declarationSigning" -> Json.obj(
+            "confirmInformationDeclaration" -> declaration,
+            "declarationCapacity" -> applicantDetails.roleInBusiness.toString
+          ),
+          "applicant" -> jsonObject(
+            "roleInBusiness" -> applicantDetails.roleInBusiness.toString,
+            "name" -> formatName(applicantDetails.name),
+            optional("previousName" -> applicantDetails.changeOfName.map(formatFormerName)),
+            "currentAddress" -> formatAddress(applicantDetails.currentAddress),
+            optional("previousAddress" -> applicantDetails.previousAddress.map(formatAddress)),
+            "communicationDetails" -> jsonObject(
+              optional("email" -> applicantDetails.contact.email),
```

Comment:
Looking at the confluence JSON email should be emailAddress and tel should be telephone. Looks like they might have missed off mobile though so one for us to check with them once we're done